### PR TITLE
Presenting: The FS .40 "Voodoo"

### PR DESCRIPTION
**Pull request by maikilangiolo**
## About The Pull Request
image
Comfortable, reliable. Tactical. The Voodoo is the new sidearm given to IH non-coms, and while it may not match the hand cannon their commanders get, it still leaves a nasty bite. The gunny can find it in his locker, or if you have the money to waste, ~~it replaced the mandella in the Bill vendor (the mandella is a nasty gun that probably should not have been left in bill's after the buffs).~~ Ok i lied, it didnt replace the mandella. but also the mandella price was fixed because it costed onl 1800 and the illegal beacon price is 4600. BONUS: while I was at it, the fucking bill vendor no longer seels the gold paint disk at 7000, which is a pain in the ass because you still had to find gold and print the paint yourself, but sells individual paint at 2000 (paint is valued at 1600, this feels like a fine markup considering its just cosmetic and limited to a few guns only).

Seeking feedback on how to make this gun cool and not too good, I asked @ErisMeleeEnjoyer to test it. In his words "The lamia is more powerful and controllable, doing more damage, but the Voodoo is more comfortable to shoot". I think this was a success overall. 
Also, the Avaralasa got a slight damage buff, since it has some hefty recoil, and it is the captain's own gun.

EDIT: **sprite was taken from Sojourn, and modified to the eris palette (and sprite size)**. The in hands are a modified Lamia in hands.

## Why Do You Bloat the Game With Guns?
Extending our .40 lineup since it was lacking, gave the

## Changelog
:cl:
add: added a new .40 pistol, the Voodoo. Available to the IH GySGT or in bill's vendor
balance: the avarasala damage mod changed 1.3 -> 1.35
/:cl:

### CODE REVIEW

**Comment by maikilangiolo** [comment]
also approved by @Firefox13 unless he gaslighted to troll me (please just leave comments to say you approve or deny on the PRs)

**Comment by ErisMeleeEnjoyer** [comment]
I'm ErisMeleeEnjoyer, and I approve of this gun.

**Comment by SirRichardFrancis** [comment]
Sprite has flashlight attached, is there flashlight functionality, Mike?

**Comment by maikilangiolo** [comment]
> Sprite has flashlight attached. Is there flashlight functionality, Mike?

image
I dont know, does the olivaw? (no, it doesnt)

**Comment by SirRichardFrancis** [comment]
>I dont know, does the olivaw? (no, it doesnt)

Maybe it should get some, idk. Instead of yet another 0.05 damage, recoil, AP, delay, etc. multiplier change.

**Comment by ErisMeleeEnjoyer** [comment]
adding working flashlights for guns is kind of out of scope for this PR

**Comment by maikilangiolo** [comment]
can be done in the future tbqh. would need to code a weaker flashlight to attach on your gun, or which is integrated already on some guns, like the olivaw. instead of fiddling with cells, you get a weaker flashlight to compensate

**Comment by ErisMeleeEnjoyer** [comment]
yeah, nuclear undisassembleable flashlights and whatnot.

**Comment by ValoTheValo** [comment]
Clearly we dont have enough IH guns

**Comment by maikilangiolo** [comment]
> Clearly we dont have enough IH guns

haha it was firefox who told me to give it to gunny

**Comment by SirRichardFrancis** [comment]
> haha it was firefox who told me to give it to gunny

Firefox to CeUvi:
>Yeah... No... There is a gun ban, can't have new guns... Those are the rules, you see... Sorry...

Firefox to Mike and Humon:
>Another gun? To IronHammer it goes, lmao.

**Comment by ErisMeleeEnjoyer** [comment]
Maik is simply built different.

**Comment by maikilangiolo** [comment]
I charmed firefox, and we became lovers

File: code/modules/projectiles/guns/projectile/pistol/voodoo.dm
Comment (Firefox13) [inline code/modules/projectiles/guns/projectile/pistol/voodoo.dm:4]:
```suggestion
			Renowed for its reliability and quick, two-round bursts. Uses .40 Magnum rounds."
```

File: code/modules/projectiles/guns/projectile/pistol/voodoo.dm
Comment (Firefox13) [inline code/modules/projectiles/guns/projectile/pistol/voodoo.dm:28]:
```suggestion
		list(mode_name="2-round bursts", mode_desc="Not quite the Mozambique method", burst=2, fire_delay=0.2, move_delay=4,    	icon="burst")
```

**Comment by TheShown911** [comment]
AAAAAAAAAAAA MORE FUCKING GUNS I AM GOING INSANE AAAAAAAAAAAAAAAAAAAA

**Comment by ErisMeleeEnjoyer** [comment]
merg his funy gun

**Comment by ErisMeleeEnjoyer** [comment]
okay, now merge it

**Comment by DiaFRAME444** [comment]
This is a nice gun, seems ready for merge.

**Comment by Humonitarian** [comment]
Alright, I approve this balance-wise.
I find adding even more guns at this stage questionable, but that isn't up to me to decide.

**Comment by ValoTheValo** [comment]
this gun is really good. abit too good. when i tested it, it had a faster TTK than the other .40 pistols and has really low recoil

**Comment by Humonitarian** [comment]
Resident balance coder, Voodoo possesses too much DPS against unarmoured targets, and too much penetration, allowing it to reliably penetrate walls, people, and maik's mom.

**Comment by maikilangiolo** [comment]
Penetration is just your fault, Humon. It's lower than the .40 revolvers, and if it's any lower then .35 pistols will just be better. 

As for DPS, Recoil is 0.6 (vs lamia 0.4 and avarasala .8). Will increase to .8 which is hefty and should reliably place it under the lamia (higher per shot damage and half recoil) and avarasala (higher damage).

**Comment by Humonitarian** [comment]
> Penetration is just your fault, Humon. It's lower than the .40 revolvers

Factually wrong 

> As for DPS, Recoil is 0.6 (vs lamia 0.4 and avarasala .8). Will increase to .8 which is hefty and should reliably place it under the lamia (higher per shot damage and half recoil) and avarasala (higher damage).

That solves the recoil, not the damage output from firerate

**Comment by maikilangiolo** [comment]
It does. You won't be able to double tap without pausing to shoot. The lamia will outshoot since it has essentially no recoil. 
As for penetration, it's lower than the deckard and mateba, higher than miller (because of miller design) and the same as the consul.

**Comment by Humonitarian** [comment]
> It does. You won't be able to double tap without pausing to shoot. The lamia will outshoot since it has essentially no recoil. As for penetration, it's lower than the deckard and mateba, higher than miller (because of miller design) and the same as the consul.

Is lamia's pen actually worse than every gun other than miller?

**Comment by maikilangiolo** [comment]
Increase it in another PR, it's a separate change. It has near zero recoil and revolver damage but with a magazine. If you want to make it better than revolvers, go ahead.

**Comment by DiaFRAME444** [comment]
I don't think increasing this thing's recoil by 0.2 will fix its DPS, specially given the multiplier is for handgun recoil.

**Comment by maikilangiolo** [comment]
I literally can't test it, it's why it's getting test merged. Kedgo tested it and it was worse than the lamia already, so the nerf is only because of a single feedback comment (and the other mentioning "broken penetration" when Humon himself broke it).

**Comment by SirRichardFrancis** [comment]
> Kegdo

> Tested

image

**Review by Humonitarian — CHANGES_REQUESTED** [review]
Increase the burst delay to at least 150% of default revolver fire delay, then it has my balance approval.

**Comment by maikilangiolo** [comment]
What the fuck is the default revolver delay

**Comment by Iglpigl** [comment]
> Increase the burst delay to at least 150% of default revolver fire delay, then it has my balance approval.

Fortunately you are not a maintener and your words hold 0 value.

**Review by Firefox13 — CHANGES_REQUESTED** [review]
Increase the burst delay to at least 150% of default revolver fire delay, then it has Humon's balance approval.

**Comment by Firefox13** [comment]
There is a lot more staff roles than maintainer.

**Comment by maikilangiolo** [comment]
Likely fucked up cause mobile

**Comment by Humonitarian** [comment]
All seems good, it has my approval @Firefox13

File: code/game/machinery/vending.dm
Comment (Humonitarian) [inline code/game/machinery/vending.dm:1679]:
```suggestion
					/obj/item/gun/projectile/voodoo = 2400,
```

File: code/game/machinery/vending.dm
Comment (Humonitarian) [inline code/game/machinery/vending.dm:1680]:
```suggestion
					/obj/item/gun/projectile/mandella = 3700,
```

**Pull request merged**
